src/content/jsonContent.ts

```diff
@@ -7,7 +7,7 @@ export class JsonContent extends HttpContent {
 
   constructor(content: any);
   constructor(content: any, mediaType: string);
-  constructor(content: any, private mediaType: string = DEFAULT_MEDIA_TYPE) {
+  constructor(content: any, mediaType: string = DEFAULT_MEDIA_TYPE) {
```

Comment:
I think `mediaType` should not be an attribute since it's only purpose is to be appened to the `headers` object